content/08.transmission.md

```diff
@@ -68,6 +68,16 @@ Data and empirically determined biological mechanisms inform models, while model
 Many current efforts to model R~t~ have led to tools that assist the visualization of estimates in real time or over recent intervals [@url:https://epiforecasts.io/covid; @url:https://rt.live].
 While these may be valuable resources, it is important to note that the estimates arise from models containing many assumptions and are dependent on the quality of the data they use, which varies widely by region.
 
+### Molecular Signatures and Transmission
+
+Genetic variation in SARS-CoV-2 has been used to elucidate patterns over time and space.
+Numerous mutations [@doi:10.1186/s12967-020-02344-6; @doi:10.1093/nsr/nwaa036; @doi:10.1016/j.meegid.2020.104351] observed in individual SARS-CoV-2 genome sequences can be used to trace transmission patterns and have provided insight during outbreak investigations. <!--move citations to later in sentence?-->
+Similar mutations observed in several patients may indicate that the patients belong to the same transmission group.
+The tracking of SARS-CoV-2 mutations is recognized as an essential tool for controlling outbreaks that may provide valuable insights into the paths of SARS-CoV-2's spread [@doi:10/d5mg].
+Several studies used phylogenetic analysis to determine the source of local COVID-19 outbreaks in Connecticut (USA), [@doi:10.1016/j.cell.2020.04.021], the New York City area (USA) [@doi:10.1126/science.abc1917], and Iceland [@doi:10.1056/NEJMoa2006100].
+There is an ongoing effort to collect SARS-CoV-2 genomes throughout the COVID-19 outbreak, and as of August 7, 2020, more than 78,000 genome sequences have been collected from patients. <!-- TODO: consider auto-updating this -->
+The sequencing data can be found at GISAID [@https://www.gisaid.org/], NCBI [@https://www.ncbi.nlm.nih.gov/sars-cov-2/], and COVID-19 data portal [@https://www.covid19dataportal.org/].
+
 ### Factors Influencing Susceptibility
 
 <!-- TODO: I think this section could use a revisiting. I think we should deal with the male sex and older age portion in an initial paragraph. After that, it looks like the racial burdens are likely to be attributable in large part to systemic racism - incarceration, wage discrimination, housing discrimination, and more - both now and in the past. Right now this jumps around b/w groups and presents discrimination as the basis in disparities as a possibility but not necessarily a likely contributor. I also worry that it makes many of these things research questions, even though one of the resources we cite ( 10.1093/tbm/ibaa055 ) includes policy recommendations. In short, it feels like we list a bunch of proximal causes but don't tie things back to the overarching likely cause. Finally, the way genetic factors are brought up makes me worry that someone might have, as a takeaway, that genetics may play a driving role in disparities at this level. Discussed in https://github.com/greenelab/covid19-review/issues/610 -->
```

Comment:
Maybe this is something to tackle in the next round? Invite some experts in this area on board? I think it is still evolving, but certainly important to include. A mention now and expand later maybe?